frontend/src/app/main/ui/export.cljs

```diff
@@ -84,55 +88,57 @@
                 (cond
-                  all-checked? [:span {:class (stl/css-case :checkbox-icon2 true
+                  all-checked? [:span {:class (stl/css-case :checkobox-tick true
                                                             :global/checked true)} i/tick-refactor]
-                  all-unchecked? [:span {:class (stl/css-case :checkbox-icon2 true
+                  all-unchecked? [:span {:class (stl/css-case :checkobox-tick true
                                                               :global/uncheked true)}]
-                  :else [:span {:class (stl/css-case :checkbox-icon2 true
+                  :else [:span {:class (stl/css-case :checkobox-tick true
                                                      :global/intermediate true)} i/remove-refactor])]]
-              [:div {:class (stl/css :selection-title)} (tr "dashboard.export-multiple.selected"
-                                                            (c (count enabled-exports))
-                                                            (c (count all-exports)))]]
+              [:div {:class (stl/css :selection-title)}
+               (tr "dashboard.export-multiple.selected"
+                   (c (count enabled-exports))
+                   (c (count all-exports)))]]
              [:div {:class (stl/css :selection-wrapper)}
               [:div {:class (stl/css-case :selection-list true
                                           :selection-shadow (> (count all-exports) 8))}
-              (for [[index {:keys [shape suffix] :as export}] (d/enumerate @exports)]
-                (let [{:keys [x y width height]} (:selrect shape)]
-                  [:div {:class (stl/css :selection-row)}
-                   [:button {:class (stl/css :selection-btn)
-                             :on-click #(on-toggle-enabled index)}
-                    [:span {:class (stl/css :checkbox-wrapper)}
-                     (if (:enabled export)
-                       [:span {:class (stl/css-case :checkbox-icon2 true
-                                                    :global/checked true)} i/tick-refactor]
-                       [:span {:class (stl/css-case :checkbox-icon2 true
-                                                    :global/uncheked true)}])]
-
-                    [:div {:class (stl/css :image-wrapper)}
-                     (if (some? (:thumbnail shape))
-                       [:img {:src (:thumbnail shape)}]
-                       [:svg {:view-box (dm/str x " " y " " width " " height)
-                              :width 24
-                              :height 20
-                              :version "1.1"
-                              :xmlns "http://www.w3.org/2000/svg"
-                              :xmlnsXlink "http://www.w3.org/1999/xlink"
+               (for [[index {:keys [shape suffix] :as export}] (d/enumerate @exports)]
+                 (let [{:keys [x y width height]} (:selrect shape)]
```

Comment:
When you create elements in a `for` you need to add the `key` otherwise React will complain